apps/src/applab/designElements/library.js

```diff
@@ -113,9 +113,10 @@ module.exports = {
 
   /**
    * @param {HTMLElement} element
+   * @param {boolean?} allowUnknown If true, we won't throw on unknown element types
    * @returns {string} String representing elementType
    */
-  getElementType: function (element) {
+  getElementType: function (element, allowUnknown) {
```

Comment:
Good catch. It was intended for `getInternalPropertyInfo`. Interestingly, with html sanitization in place I think it becomes impossible to actually add elements that we won't be able to discover an element type for. That said, I'll still add this param (and use is properly) in case things get through the cracks, and because we're not sanitizing on load atm.